Basic/DistributedAuthoritySocialHub/Assets/Scripts/UI/IngameUI/TextChatManager.cs

```diff
@@ -118,7 +116,7 @@ void BindSessionEvents(bool doBind)
             }
         }
 
```

Comment:
Not related to this line specifically, but after the merge, somewhere in here, when the text tray button is pressed, we'll have to invoke [this](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/d463e6a8dde1720bef665d50a9aba581b4bfa9d4/Basic/DistributedAuthoritySocialHub/Assets/Scripts/Input/InputSystemManager.cs#L98) to enable/disable gameplay input.

Reply:
thanks for the hint, will implement in a follow-up. I have to change the Textchat to mobile anyway. will address this in a follow up

Reply:
Sounds good. Let's create a ticket!